ci/prow/periodic_config.go

```diff
@@ -383,10 +383,10 @@ func generateIssueTrackerPeriodicJob(jobName, labelFilter, updatedTime, comment
 	data.Base.Command = "/app/robots/commenter/app.binary"
 
 	data.Base.Args = []string{
-		fmt.Sprintf(`|-
+		`|-
```

Comment:
Funny, can't you just use \n instead of the multi-line string for a single line?

Reply:
I tried but the indentation was mysteriously incorrect :(